src/main/java/core/basesyntax/ReadData.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReadData {
+    public String getDataFromFile(String fromFile) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFile))) {
+            StringBuilder data = new StringBuilder();
+            for (Object line : bufferedReader.lines().toArray()) {
```

Comment:
you can use `toList()` intead of `toArray()` so that you won't have `Object` as a loop variable